frontend/nexus/utils.ts

```diff
@@ -145,7 +137,7 @@ export const generateExcerpt = (
 export const readTime = (text: string): number => {
   const numWords = text.split(' ').length
 
-  return Math.round(numWords / 200)
+  return Math.round(numWords / 200) || 1
```

Comment:
Changes in file are from prettier except this line

Reply:
@robin-macpherson Since this gets saved to the DB, it means that future new posts or edited posts will get the "minimum 1 min. read time", but previous posts by alpha testers will still see 0 min. Should I also add an `|| 1` on the frontend for the few alpha tester posts, or just keep it like this for simplicity?

Reply:
> Changes in file are from prettier except this line

I've honestly been finding the prettier stuff pretty annoying to be honest 😕

Reply:
> @robin-macpherson Since this gets saved to the DB, it means that future new posts or edited posts will get the "minimum 1 min. read time", but previous posts by alpha testers will still see 0 min. Should I also add an `|| 1` on the frontend for the few alpha tester posts, or just keep it like this for simplicity?

I was actually assuming that for the moment this would just be a simple frontend change for now since the primary urgency is just presentational and we may want to think more carefully about how we want to represent this in the DB

Reply:
(for example, we may still want the real result of the calculation represented in the data that we store. We may want to run analysis on how many posts are created with 0 minutes, and this also currently helps us identify posts that are impacted by #208

Reply:
> > Changes in file are from prettier except this line
> 
> I've honestly been finding the prettier stuff pretty annoying to be honest 😕

Really?? I don't think I could live without it now, but I think we should add [some infra via husky](https://prettier.io/docs/en/precommit.html) or something that way files don't get saved _without_ prettier, causing larger file changes when someone edits that file

Reply:
> > @robin-macpherson Since this gets saved to the DB, it means that future new posts or edited posts will get the "minimum 1 min. read time", but previous posts by alpha testers will still see 0 min. Should I also add an `|| 1` on the frontend for the few alpha tester posts, or just keep it like this for simplicity?
> 
> I was actually assuming that for the moment this would just be a simple frontend change for now since the primary urgency is just presentational and we may want to think more carefully about how we want to represent this in the DB

I agree! will update